### ISSUE TITLE
openSUSE Leap menu entries

### ISSUE THREAD

**Issue by dbouras** [cross-platform/icloud-for-linux#44]
The desktop entries installed under /var/lib/snapd/desktop/applications don't have a "Categories=" key. According to the [Desktop entry spec](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html) this is not required, but most distros/desktop environments (openSUSE Leap/KDE being one such case) use it to figure out where to load it on the Application Menu.

The solution is of course simple. A unified patch for e.g. icloud-for-linux_contacts.desktop is as follows (all other entries can be modified in the same way):

```
--- orig/icloud-for-linux_contacts.desktop	2021-07-12 13:07:19.285906406 -0700
+++ icloud-for-linux_contacts.desktop	2021-07-18 12:57:05.356996683 -0700
@@ -5,6 +5,7 @@
 GenericName=iCloud Contacts
 Comment=iCloud Contacts
 Type=Application
+Categories=Office
 Icon=/snap/icloud-for-linux/216/meta/gui/contacts.png
 Exec=env BAMF_DESKTOP_FILE_HINT=/var/lib/snapd/desktop/applications/icloud-for-linux_contacts.desktop /snap/bin/icloud-for-linux.contacts
 StartupWMClass=icloud-for-linux.contacts
```

I chose "Office" because it's one of the ["Main" categories](https://specifications.freedesktop.org/menu-spec/latest/apa.html) and it made the most sense to me.
Many thanks for contributing such a useful project!

**Comment by MarcusTomlinson** [comment]
This should be fixed in v0.12 now. Please run `snap refresh icloud-for-linux`.

**Comment by dbouras** [comment]
Works like a charm now - thanks!
